apps/web/src/app/blog/page.tsx

```
@@ -1,71 +1,43 @@
+          </p>
         </div>
       </section>
-    </div>
+      <div className="min-h-screen grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 p-4">
+        {posts.map((post) => (
+          <BlogCard
+            post={{
+              ...post.metadata,
+              slug: post.slug,
+            }}
+          />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)